tests/potential/test_potential.py

```diff
@@ -431,6 +479,14 @@ def test_zeros(self):
         np.testing.assert_allclose(u, u_copy)
         self.assertEqual(s, False)
 
+        #test with 1-element array
+        r = [0.2]
+        r_copy, u, s = p._zeros(r)
+        u_copy = np.zeros(1)
+        np.testing.assert_allclose(r_copy, r)
```

Comment:
For consistency with next line:
```suggestion
        np.testing.assert_allclose(r, r_copy)
```